devops/ansible/roles/php70/vars/main.yml

```diff
@@ -9,7 +9,7 @@ debug_mode: false
 # allow_url_fopen
 #   Default Value: On
 php_allow_url_fopen: "Off"
-php_disable_functions: "exec,passthru,shell_exec,system,proc_open,popen"
+php_disable_functions: "exec,passthru,shell_exec,system,popen"
```

Comment:
Ok. If needed, include a comment and link explaining why. Limit scope as much as possible.